src/react/components/common/errorHandler/errorHandler.tsx

```diff
@@ -103,29 +102,40 @@ export class ErrorHandler extends React.Component<IErrorHandlerProps> {
                 message: reason.message,
                 title: reason.name,
             };
+        } else {
+            appError = {
+                title: strings.errors.unknown.title,
+                errorCode: ErrorCode.Unknown,
+                message: this.getUnknownErrorMessage(error),
+            };
         }
+        this.props.onError(appError);
+    }
 
-        if (!appError) {
-            appError = new AppError(ErrorCode.Unknown, "Unknown Error occurred");
+    private getUnknownErrorMessage(e) {
+        if (Env.get() !== "production") {
```

Comment:
I'm assuming when we do a prod build `create-react-app` sets node_env to "production" ?

Reply:
That's what I understood. I could be wrong, but I'll look into this further

Reply:
Verified this with a release version.